src/test/java/com/ably/kafka/connect/ChannelSinkTaskIT.java

```diff
@@ -56,55 +55,236 @@ public void setup() throws Exception {
         appSpec = AblyHelpers.createTestApp();
 
         // setup Connect cluster with defaults
-        connect = new EmbeddedConnectCluster.Builder().build();
+        connectCluster = new EmbeddedConnectCluster.Builder().build();
 
         // start Connect cluster
-        connect.start();
-        connect.assertions().assertAtLeastNumWorkersAreUp(NUM_WORKERS, "Initial group of workers did not start in time.");
+        connectCluster.start();
+        connectCluster.assertions().assertAtLeastNumWorkersAreUp(NUM_WORKERS, "Initial group of workers did not start in time.");
 
         // connect Ably client
         ablyClient = AblyHelpers.realtimeClient(appSpec);
     }
 
     @AfterEach
     public void close() {
-        connect.stop();
+        connectCluster.stop();
         ablyClient.close();
         AblyHelpers.deleteTestApp(appSpec);
     }
 
+    //channel name tests
     @Test
-    public void testMessagePublish() throws Exception {
-        // create a test Kafka topic
-        connect.kafka().createTopic("kafka-connect-ably-test");
+    public void testMessagePublish_channelExistsWithStaticChannelName() throws Exception {
+        final String channelName = "test-channel";
+        // topic1
+        final String topic = TOPICS.split(",")[0];
```

Comment:
This is repeated in every test so I'd suggest creating another field in the class called `DEFAULT_TOPIC` or `FIRST_TOPIC` and assigning that value to it :wink: